libpod/doc.go

```diff
@@ -0,0 +1,11 @@
+// The libpod library is not stable and we do not support use cases outside of
```

Comment:
Maybe we should say "At present"? I can see us making a stable interface at some point in the future

Reply:
If we do this we can change the comment. I would not give people false hopes and this would mean users asking when it will happen but we don't even know if it will happen.

Reply:
I agree, if we decide to support libpod as a library in the future we can change the docs.